spec/kittikrb/cursor_spec.rb

```diff
@@ -122,15 +122,241 @@ def current_rendered_buffer
       end
     end
 
-    describe '#move_to' do
+    context 'move methods' do
       def current_coords
         [cursor.instance_variable_get(:@x), cursor.instance_variable_get(:@y)]
       end
 
-      it 'should properly set absolute position of cursor' do
+      before do
         expect(current_coords).to eq [0, 0]
-        expect(cursor.move_to(5, 10)).to be_instance_of described_class
-        expect(current_coords).to eq [5, 10]
+      end
+
+      describe '#move_to' do
+        it 'should properly set absolute position of cursor' do
+          expect(cursor.move_to(5, 10)).to be_instance_of described_class
+          expect(current_coords).to eq [5, 10]
+        end
+      end
+
+      describe '#move_by' do
+        it 'should properly set positive offset position of cursor' do
+          expect(cursor).to receive(:right).with(7)
+          expect(cursor).to receive(:down).with(3)
+          expect(cursor.move_by(7, 3)).to be_instance_of described_class
+        end
+
+        it 'should properly set negative offset position of cursor' do
+          expect(cursor).to receive(:left).with(2)
+          expect(cursor).to receive(:up).with(4)
+          expect(cursor.move_by(-2, -4)).to be_instance_of described_class
+        end
```

Comment:
But there is no tests for changing cursor position at all. You should check `current_coords` after calling `move_by`.